CMakeLists.txt

```diff
@@ -34,42 +28,20 @@ if(POLICY CMP0012)
   cmake_policy(SET CMP0012 NEW)
 endif()
 
-find_package(OpenMP)
+if(OPENMP)
+  find_package(OpenMP REQUIRED COMPONENTS Fortran)
+endif()
+
 find_package(LAPACK)
```

Comment:
Which utility requires LAPACK? Since it is not required, do we handle the case when it is not found?

Reply:
Ok, I removed that